src/java.desktop/share/classes/javax/swing/JTabbedPane.java

```diff
@@ -722,10 +722,10 @@ public void setSelectedComponent(Component c) {
      * @param component the component to be displayed when this tab is clicked.
      * @param tip the tooltip to be displayed for this tab
      * @param index the position to insert this new tab
-     *       ({@code > 0 and <= getTabCount()})
+     *       {@code (index >= 0 && index <= tab count)}
      *
```

Comment:
Now you've made it more like code, except for less like code by changing getTabCount() to tab count.

I suggest
* @param index the position to insert this new tab
* where {@code (index >= 0 && index <= getTabCount())}

Reply:
Updated @prrace .......